apps/extension/src/background/web-workers/submit-transfer-web-worker.ts

```diff
@@ -0,0 +1,37 @@
+import { Sdk } from "@anoma/shared";
+import { init as initShared } from "@anoma/shared/src/init";
+import { IndexedDBKVStore } from "@anoma/storage";
+import { fromBase64 } from "@cosmjs/encoding";
+import { SDK_KEY } from "background/keyring";
+import { KVPrefix } from "router";
+import { INIT_MSG } from ".";
+
+const DEFAULT_URL =
```

Comment:
Minor thing, but we could simply pull the URL as specified in the `namada` chain config in `@anoma/chains`, which will handle any overrides in the `.env` as well